src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/utilities/Configs.scala

```diff
@@ -125,3 +125,8 @@ object HealthConfig {
   lazy val yarnConnectionTestEndpoint: String =
     Configs.conf.getString("health.yarnConnection.testEndpoint")
 }
+
+object ApplicationConfig {
+  lazy val maximumNumberOfWorkflowsInBulkRun: Int =
```

Comment:
I think it makes no sense to use `lazy` here. I don't know why I used it in the `HealthConfig`

Reply:
ok